modules/editor/message/MessageEditor.tsx

```diff
@@ -69,11 +88,38 @@ export function MessageEditor(props: MessageEditorProps) {
           Add Embed
         </PrimaryButton>
       </div>
-      <div>
+      <InputField
+        id={`_${message.id}_reference`}
+        label="Message Link"
+        placeholder="https://discord.com/channels/..."
+        error={form.field("reference").error}
+        {...form.field("reference").inputProps}
+      />
+      <Message
+        content={
+          "*When a message link is set, it allows you to edit previously " +
+          "sent messages.*"
```

Comment:
Rewording this, please accept I don't want to deal with merge conflicts.
```suggestion
          "*When a message link is set, pressing submit or edit will edit the" +
          " message sent inside of Discord. To load a message sent in Discord, use" +
          " the bot's 'link' command.*"
```